python/revng/internal/cli/_commands/import_idb.py

```diff
@@ -75,6 +75,18 @@ def run(self, options: Options):
                 options,
             )
 
+            run_revng_command(
```

Comment:
I think you can do all of these invocations in a single shot, can you?

Reply:
No reason to even think about it, if we wrap all of it in `fixModel`, this will have to be a single invocation anyway.

Reply:
Done.